jboss/container/prometheus/8.2/module.yaml

```diff
@@ -0,0 +1,27 @@
+schema_version: 1
+
+name: jboss.container.prometheus
+version: '8.2'
+description: ^
+  Provides support for configuring Prometheus.  Basic usage is
+  JAVA_OPTS="$JAVA_OPTS $(source $JBOSS_CONTAINER_PROMETHEUS_MODULE/prometheus-opts; get_prometheus_opts)"
+
+envs:
+- name: JBOSS_CONTAINER_PROMETHEUS_MODULE
```

Comment:
This is dropped from most of the scripting above, but its something we've documented as API -- should we support this in the scripting or mark it as deprecated here or something?

Reply:
The JAR is no longer placed under that location, but the prometheus-opts file and the jmx-exporter-config.yaml are still in there. I thought the JBOSS_CONTAINER_PROMETHEUS_MODULE variable was an informational env var rather than a config knob. Could people have been using it the other way?

Reply:
Not entirely sure, the usage in prometheus-opts suggest that they could have. It is possible I suppose that someone could have needed to run a specific version of prometheus and then they could use this mechanism. But I don't know how likely this is.

Reply:
I think it's really unlikely. and if they did, I wonder where in the image they would put it - a different place (and therefore need to change that  variable) or over the top of the one that we supplied?